test-data/unit/stubgen.test

```diff
@@ -355,12 +396,17 @@ __all__ = ['p']
 import urllib.parse as p
 
 [case testExportModuleInPackageUnsupported_import]
-import urllib.parse
 __all__ = ['urllib']
 [out]
 # Names in __all__ with no definition:
 #   urllib
 
+[case testExportModuleInPackage_import]
+import urllib.parse
+__all__ = ['urllib']
+[out]
+import urllib.parse
```

Comment:
Yes, then ignore this comment (or add another test where this produce an error, like ``from urllib import parse``)